### ISSUE TITLE
Libero dataset action space in cartesian

### ISSUE THREAD

**Issue by yuwei-will-xia** [Physical-Intelligence/openpi#416]
Thanks for the work!

I am fine-tuning my own data based on eef-pose in Cartesian as the action space and compared my datasets with the libero datasets [https://huggingface.co/datasets/openvla/modified_libero_rlds/blob/main/libero_goal_no_noops/1.0.0/features.json]

It turns out the libero action(7-dim)/state(8-dim) data are all in eef pose, which is not working well with the pi0 model`s data transforms between delta/absolute action functions(you can not just subtract the rotation value)

Does the model support Cartesian action space? If so, how can I set the Cartesian action in delta to adapt to the model?

Thanks!

**Comment by xianjiedai** [comment]
There is an related issue: https://github.com/Physical-Intelligence/openpi/issues/347.

**Comment by kpertsch** [comment]
Hi!

I think you can just remove the DeltaAction transforms (ie [this](https://github.com/Physical-Intelligence/openpi/blob/36dc3c037eb8a3921be9ecb94369d60cbf56f58f/src/openpi/training/config.py#L305) code block, see my TODO) to train on your data. I would expect the model to still be able to learn if you finetune it on some in-domain data.

Hope this helps! Will close this issue for now, feel free to reopen if necessary!
